### PR TITLE
Error message corrected

### LINKED ISSUE #626: Wrong error message displayed.
**Actual Behaviour**
When visited to https://accounts.susi.ai/resetpass without logging in, wrong error message is displayed.
capture

**Expected Behaviour**
Instead of "Invalid Token", "To reset password visit Homepage" should be displayed.

**Steps to reproduce it**

1) Visit https://accounts.susi.ai/resetpass without logging to your account.

**Would you like to work on the issue?**
Yes,I would like to work on this.

### PR DESCRIPTION
**Pull request by plxity**
Fixes #626 

Changes: Correct error message is displayed when someone visits reset password route.

Surge Deployment Link: https://pr-629-fossasia-susi-accounts.surge.sh

### CODE REVIEW

**Comment by plxity** [comment]
@anshumanv @akshatnitd @fragm3  Please review this.

File: src/components/Auth/ResetPassword/ResetPassword.react.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ class ResetPassword extends Component {
       }.bind(this),
       error: function(errorThrown) {
         let state = this.state;
-        state.msg = 'Invalid token!';
+        state.msg = 'To reset password visit Homepage!';
```
Comment (AakashMallik) [inline src/components/Auth/ResetPassword/ResetPassword.react.js]: This error message would not make sense to be very honest. Actually, if you have an invalid token ( token which was not provided by the server ) then it is a case of malicious entry. In that case, "invalid user" or 'username not found' may be a better alternative.

Comment (plxity) [reply src/components/Auth/ResetPassword/ResetPassword.react.js]: @AakashMallik Okay, So Should I change it to "Username not found"?

Comment (AakashMallik) [reply src/components/Auth/ResetPassword/ResetPassword.react.js]: @plxity IMO, yup. But ask @anshumanv before finalizing it.

Comment (plxity) [reply src/components/Auth/ResetPassword/ResetPassword.react.js]: @AakashMallik Okay, sure!

Comment (PrP-11) [reply src/components/Auth/ResetPassword/ResetPassword.react.js]:
```suggestion
        state.msg = 'Invalid request!';
```
I think this would be better.

**Comment by plxity** [comment]
@anshumanv @akshatnitd Please review this.

**Comment by plxity** [comment]
@anshumanv @akshatnitd Can this be reviewed and merged?

**Pull request merged**